sdk/storage/storage-blob/src/utils/utils.common.ts

```diff
@@ -268,7 +268,9 @@ export function appendToURLPath(url: string, name: string): string {
   path = path ? (path.endsWith("/") ? `${path}${name}` : `${path}/${name}`) : name;
   urlParsed.pathname = path;
 
-  return urlParsed.toString();
+  const normalizedUrl = new URL(urlParsed.toString());
```

Comment:
@jeremymeng the original PR was working around behavior inside `URLBuilder`, but since we're using `URL` to parse things now, do we need these changes? I'm curious if we undo this change if the test still passes.

Reply:
Yes the tests are still passing.  I undid the src changes.  Thanks for pointing out!